src/lib/Libifl/pbs_loadconf.c

```diff
@@ -1097,7 +1106,7 @@ __pbs_loadconf(int reload)
 
 	pbs_conf.loaded = 1;
 
-	if (parse_psi(psi_value ? psi_value : pbs_default()) == -1) {
+	if (parse_psi(psi_value) == -1) {
 		fprintf(stderr, "Couldn't find a valid server instance to connect to\n");
 		free(psi_value);
```

Comment:
Remove this free and other `free(psi_value)` else it will be double free as parse_psi is already freeing given value (even in error case also)